packages/fast-components-react-msft/src/select/select.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Select",
+    "description": "A select component's schema definition.",
+    "type": "object",
+    "id": "select",
```

Comment:
This ID must be changed to `@microsoft/fast-components-react-msft/select`

Reply:
done